Source/Core/DolphinLib.props

```diff
@@ -20,7 +20,6 @@
     <ClInclude Include="Common\BitSet.h" />
     <ClInclude Include="Common\BitUtils.h" />
     <ClInclude Include="Common\BlockingLoop.h" />
-    <ClInclude Include="Common\CDUtils.h" />
```

Comment:
Seems like you forgot to remove `CDUtils.cpp`. Removing it should fix the Windows build. https://github.com/dolphin-emu/dolphin/blob/0385a414a177eccc0d204ed461736e26d1316bc8/Source/Core/DolphinLib.props#L722

Reply:
Done!